packages/clerk-js/src/core/clerk.test.ts

```diff
@@ -500,6 +500,40 @@ describe('Clerk singleton', () => {
         expect(sut.setActive).toHaveBeenCalledWith({ session: null });
       });
     });
+
+    it('destroyes the current client after sign out', async () => {
+      const spyOnDestroy = jest.fn().mockImplementation(() => {
```

Comment:
⛔  this test does not test that the destroy is resetting the `Client.instance` since the destroy is being mocked.

Reply:
I don't think there a good way to test that behavior with a unit test, so I should probably write a integration test for this.